### PR TITLE
[CDAP-11761] Highlight current file in connector browser for file system, S3, GCS

### PR DESCRIPTION
**Pull request by njbriggs**
Jira: https://issues.cask.co/browse/CDAP-11761
If the current workspace is a file and that file is in view in the file browser, the file is highlighted.

image

### CODE REVIEW

**Comment by njbriggs** [comment]
E2E: https://builds.cask.co/browse/IT-UPD2204

File: cdap-ui/app/cdap/components/FileBrowser/index.js [hunk 1]
```diff
@@ -464,7 +478,11 @@ export default class FileBrowser extends Component {
 
         <div className="content-body clearfix">
           {displayContent.map((content) => {
-            return this.renderRow(content);
+            let isRowSelected = false;
+            if (inWorkspaceDirectory && content.name === this.getFileName()) {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/FileBrowser/index.js]: Can we do this check directly in the `renderRowContent`? Seems like ```this.state.path === this.getFilePath() && content.name === this.getFileName()``` should be available in the specific function. 

Avoids passing this prop down multiple levels.

**Review by ajainarayanan — COMMENTED** [review]
@njbriggs Also we need to do similar thing to other connections as well (gcs, bigquery, db, s3 etc.,) right? If we are adding for file browser we might as well do it across all browsers.

File: cdap-ui/app/cdap/components/FileBrowser/index.js (hunk 1 above)
Comment (njbriggs) [reply cdap-ui/app/cdap/components/FileBrowser/index.js]: You can do the full check there, but my concern was that perform the path check once per row was excessive. If you think it's not a concern, I can make the change.

**Comment by njbriggs** [comment]
@ajainarayanan Yes, was planning to ask about the other connectors. The original bug was written for files but it does make sense to do this for all connectors.

File: cdap-ui/app/cdap/components/FileBrowser/index.js (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/FileBrowser/index.js]: I think that should be ok. Its a constant time operation (1 check).

**Comment by njbriggs** [comment]
Updated to handle: file system, S3, and GCS. This will need to be cherry-picked to 6.3.

**Comment by njbriggs** [comment]
New E2E test link: https://builds.cask.co/browse/IT-UPD2208

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Pull request merged**